src/basic/glyph-util.c

```diff
@@ -139,3 +141,74 @@ const char *special_glyph(SpecialGlyph code) {
         assert(code < _SPECIAL_GLYPH_MAX);
         return draw_table[code >= _SPECIAL_GLYPH_FIRST_EMOJI ? emoji_enabled() : is_locale_utf8()][code];
 }
+
+const char *special_glyph_utf8(SpecialGlyph code) {
```

Comment:
don't duplicate the entire list, instead just add a parameter to the existing function that overrides the check

Reply:
Okay sir.
That's more efficient.

Reply:
@bluca
I'm considering changing the prototype of the function from 
```const char *special_glyph(SpecialGlyph code) _const_;```
to
```const char *special_glyph(SpecialGlyph code, bool use_fallback) _const_;```

But there'll be conflicts with all existing instances of its use.

What do you advise sir?

Reply:
call it `bool force_utf` instead, and then update all existing calls to simply pass `/* force_utf= */ false`

Reply:
> call it `bool force_utf` instead, and then update all existing calls to simply pass `/* force_utf= */ false`

😂😂😂😂😂
Awesome.

Can't be more than 100 files I have to edit.
😂😂😂

Reply:
Let me grab lunch and get started sir.
Now that I'm pushing this, would you help me get unstuck on the journal?
Thank you!!!

Reply:
you should be able to programmatically fix it using `sed` or similar tools, for example `sed -i "s|special_glyph(\(.*\))|special_glyph(\1, /* force_utf= */ false)|g" path/to/source.c` or so should work

Reply:
> you should be able to programmatically fix it using `sed` or similar tools, for example `sed -i "s|special_glyph(\(.*\))|special_glyph(\1, /* force_utf= */ false)|g" path/to/source.c` or so should work

Wow...
I'll certainly try.

Reply:
Sed works, but I'm still having to manually check. Some parenthesis issues, and I initially did ```find -name "*.c"``` the function also appears in some headers.
About 6 files done so far.